inbox/transactions/actions.py

```diff
@@ -222,9 +222,9 @@ def _process_log(self):
                 else:
                     namespaces_to_process = random.sample(namespace_ids,
                                                           NUM_PARALLEL_ACCOUNTS)
-                self.log.info('Syncback namespace_ids count', shard_id=key,
-                              process=self.process_number,
-                              num_namespace_ids=len(namespace_ids))
+                #self.log.info('Syncback namespace_ids count', shard_id=key,
```

Comment:
Oh I commented those because they spam the console and make it super annoying to debug API requests.